### PR TITLE
Create ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3470](https://cumulocity.atlassian.net/browse/DM-3470)
Original PR: [6079](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6079)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3470
+version: 1018.503.77
+---
+Fix software list schema definition for properties library
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md:17]: The Claude AI suggested to use the following body:
```suggestion
The schema definition for the software list properties library contained an error that caused unexpected behavior when working with software list properties. This issue has been resolved by correcting the schema definition. The fix ensures that the software list properties library now functions as intended, allowing users to properly utilize and manage software list properties without encountering schema-related problems. No additional actions are required from users to benefit from this fix.
```

File: content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fix software list schema definition for properties library
```
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md:3]:
```suggestion
title: Software items are now properly shown in the Device data widget
```

File: content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md (hunk 1 above)
Comment (bozhidardedov) [inline content/change-logs/device-management/ui-c8y-1018-503-77-fix-software-list-schema-definition-for-properties-library.md:17]:
```suggestion
The properties library schema definition for the `c8y_SoftwareList` fragment did not match the actual managed object properties. Hence, when the "Software" property was selected in the "Device data" widget no data was shown. The property is now called "Software list" and displays all software items contained in the `c8y_SoftwareList` fragment.
```

**Review by BeateRixen — APPROVED** [review]

**Comment by bozhidardedov** [comment]
@BeateRixen Please note that the fix has been also backported to version 10.18.0.254

**Review by l2p- — APPROVED** [review]

**Pull request merged**